webextensions/api/permissions.json

```
@@ -174,7 +174,9 @@
                 ]
               },
               "firefox_android": {
-                "version_added": true
+                "version_added": true,
+                "partial_implementation": true,
+                "notes": "Since version 79, the UI to approve the permission request is missing (<a href='https://bugzil.la/1601420'>bug 1601420</a>)."
```

There's not an actual problem here, just some style stuff to make this more consistent with other notes.

```suggestion
                "notes": "From version 79, the user interface to approve the permission request is missing. See <a href='https://bugzil.la/1601420'>bug 1601420</a>."
```